internal/operators/cnv/config_test.go

```diff
@@ -0,0 +1,64 @@
+package cnv_test
+
+import (
+	"os"
+
+	"github.com/kelseyhightower/envconfig"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/operators/cnv"
+)
+
+var _ = Describe("CNV plugin configuration", func() {
+	Context("for GPU", func() {
+		const (
+			prefix           = "test"
+			supportedGpusKey = "TEST_CNV_SUPPORTED_GPUS"
```

Comment:
Is this one still necessary?

Reply:
Yes, this one is used in testing the decoder. 
It doesn't clash with main assisted-service variable thanks to a different prefix.